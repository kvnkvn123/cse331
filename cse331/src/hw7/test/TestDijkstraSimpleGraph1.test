# Test Dijkstra's algorithm for a simple graph

CreateGraph simple
AddNode simple a
AddNode simple b
AddNode simple c
AddNode simple d
AddNode simple e
AddNode simple f
AddNode simple g
AddNode simple h
ListNodes simple
AddEdge simple a b 2
AddEdge simple a c 1
AddEdge simple a d 4
AddEdge simple b c 5
AddEdge simple b e 10
AddEdge simple b f 2
AddEdge simple d c 2
AddEdge simple c a 9
AddEdge simple c e 11
AddEdge simple e d 7
AddEdge simple e g 1
AddEdge simple f h 3
AddEdge simple g e 3
AddEdge simple g f 2
AddEdge simple h g 1

FindPath simple a a
FindPath simple a b
FindPath simple a f
FindPath simple a h