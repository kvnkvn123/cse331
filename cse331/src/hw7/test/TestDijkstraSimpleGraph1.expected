# Test Dijkstra's algorithm for a simple graph

created graph simple
added node a to simple
added node b to simple
added node c to simple
added node d to simple
added node e to simple
added node f to simple
added node g to simple
added node h to simple
simple contains: a b c d e f g h
added edge 2.000 from a to b in simple
added edge 1.000 from a to c in simple
added edge 4.000 from a to d in simple
added edge 5.000 from b to c in simple
added edge 10.000 from b to e in simple
added edge 2.000 from b to f in simple
added edge 2.000 from d to c in simple
added edge 9.000 from c to a in simple
added edge 11.000 from c to e in simple
added edge 7.000 from e to d in simple
added edge 1.000 from e to g in simple
added edge 3.000 from f to h in simple
added edge 3.000 from g to e in simple
added edge 2.000 from g to f in simple
added edge 1.000 from h to g in simple